app/build.gradle

```diff
@@ -621,11 +624,11 @@ task printVariants {
             engine: 'geckoNightly',
             name: 'androidTest',
         ])
-        println 'variants: ' + groovy.json.JsonOutput.toJson(variants)
+        println 'variants: ' + JsonOutput.toJson(variants)
     }
 }
 
-task buildTranslationArray {
+tasks.register('buildTranslationArray') {
```

Comment:
I don't see this called from anywhere and it doesn't set dependencies but it seems to correctly fill the `SUPPORTED_LOCALE_ARRAY`: do you know why this works and why it will continue to work after lazy configuration?

I'm wondering if this has been running in the configuration phase this whole time (e.g. the code is not in `doLast`) and lazily registering it will prevent it from working.

Reply:
I think you're correct and the array is being populated as a side effect. I'll revert this line and file an issue.